src/SALib/analyze/hdmr.py

```diff
@@ -496,7 +496,7 @@ def f_test(Y, f0, Y_em, R, alpha, m1, m2, m3, n1, n2, n3, n):
         # Number of parameters of proposed model (order dependent)
         if i <= n1:
             p1 = m1        # 1st order
-        elif i > n1 and i <= (n1 + n2):
+        elif n1 < i <= n1 + n2:
```

Comment:
I would prefer the brackets around n1 + n2 be kept, as this is clearer to me.

```python
elif n1 < i <= (n1 + n2):
```

Reply:
Okay, I just added the  brackets. Thanks for your comments.